src/locale/constants.ts

```diff
@@ -0,0 +1,4 @@
+// The narrow unit display for `month` in English is `m`, which conflicts with
+// the one for `minute` (also `m`), this also goes for any locale that currently
+// falls back to English for narrow-unit display (missing localization?)
+export const MONTH_FALLBACK_LOCALES = ['en', 'ja', 'es', 'fr', 'tr']
```

Comment:
I've checked all the locales we currently have and these are the ones that currently falls back to English as of Firefox 128 (nightly)

I'd have to recheck for Chrome as they could be on different versions of ICU. Curious how it'll be for Native as well.

Reply:
Since `i18n.number` does not work as expected for Japanese period expressions, it would be better to patch `ago()` as in #3905.

Reply:
Hmm, you might be right, I was hoping I could avoid that since this approach would mean avoiding adding strings for something that the browser should already have (in its ICU data). I'll change it in just a bit

Reply:
Replaced with strings, means that JP and other locales listed there can have proper localization for relative time